src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSwitchedToSole.java

```diff
@@ -129,4 +130,18 @@ private CaseData removeApplicant2AnswersFromCase(CaseData caseData) {
 
         return caseData;
     }
+
+    private CaseData removeJurisdictionAnswersFromCase(CaseData caseData) {
+        caseData.getApplication().getJurisdiction().setApplicant1Domicile(null);
+        caseData.getApplication().getJurisdiction().setApplicant2Domicile(null);
+        caseData.getApplication().getJurisdiction().setResidualEligible(null);
+        caseData.getApplication().getJurisdiction().setConnections(null);
+        caseData.getApplication().getJurisdiction().setApplicant1Residence(null);
+        caseData.getApplication().getJurisdiction().setApplicant2Residence(null);
+        caseData.getApplication().getJurisdiction().setBothLastHabituallyResident(null);
+        caseData.getApplication().getJurisdiction().setApp1HabituallyResLastTwelveMonths(null);
+        caseData.getApplication().getJurisdiction().setApp1HabituallyResLastSixMonths(null);
+
+        return caseData;
+    }
```

Comment:
I think you can just do 
```suggestion
    private CaseData removeJurisdictionAnswersFromCase(CaseData caseData) {
        caseData.getApplication().setJurisdiction(null);
        return caseData;
    }
```

Reply:
In which case you can just add it to the removeApplicant2AnswersFromCase()